x-pack/plugin/spatial/src/internalClusterTest/java/org/elasticsearch/xpack/spatial/search/CartesianShapeQueryTestCase.java

```diff
@@ -71,8 +69,7 @@ protected Point nextPoint() {
     }
 
     protected Polygon nextPolygon() {
-        org.apache.lucene.geo.Polygon randomPoly = GeoTestUtil.nextPolygon();
-        return new Polygon(new LinearRing(randomPoly.getPolyLons(), randomPoly.getPolyLats()));
+        return ShapeTestUtils.randomPolygon(false);
```

Comment:
I had used the GeoTestUtil polygon before because I wanted one polygon in 'normal' range (small reasonable numbers) amongst all the other test polygons that were using extreme numbers. But switching back to the ShapeTestUtils is much more readible, rational, and allows the same fix for the geometry collection to apply to the single polygon.